### PR TITLE
Ansible 8.2.0: Dependencies, changelog and porting guide

### PR DESCRIPTION
**Pull request by anweshadas**

### CODE REVIEW

File: 8/porting_guide_8.rst [hunk 1]
```diff
@@ -17,82 +17,91 @@ Ansible 8 is based on Ansible-core 2.15.
 
 We suggest you read this page along with the `Ansible 8 Changelog <https://github.com/ansible-community/ansible-build-data/blob/main/8/CHANGELOG-v8.rst>`_ to understand what updates you may need to make.
```
Comment (felixfontein) [inline 8/porting_guide_8.rst:19]: Ah, and now we see another result of moving the docs/ folder out of ansible/ansible: antsibull cannot find the ansible-core porting guide anymore and include it in the Ansible porting guide... That's now a part of the ansible-documentation git repo.

Which means it just became a **lot** more important to have that repository tagged for releases as well CC @gotmax23.

Ref: https://github.com/ansible/ansible-documentation/issues/66

**Review by felixfontein — APPROVED** [review]
That the porting guide is screwed up is somewhat unfortunate; we have to fix that eventually, but I'm not sure we'll manage for today's release. (I can fix it, but I have no time today and likely not enough time the rest of this week, so delaying by 1-2 days won't help.)

**Review by mariolenz — APPROVED** [review]
Since 8.2.0 is already released, I think we should merge this.

**Review by gotmax23 — APPROVED** [review]

**Comment by gotmax23** [comment]
Thanks for working on the release during your vacation, @anweshadas!

**Pull request merged**